Tasks/BeanstalkDeployApplication/helpers/DeployApplicationTaskOperations.ts

```diff
@@ -191,22 +191,15 @@ export class TaskOperations {
                 requestEvents.StartTime = lastPrintedEventDate;
                 const responseEvent = await this.beanstalkClient.describeEvents(requestEvents).promise();
 
-                if (responseEvent.Events.length > 0) {
-                    for (let i = responseEvent.Events.length - 1; i >= 0; i--) {
-                        const event = responseEvent.Events[i];
-                        if (event.EventDate <= lastPrintedEventDate) {
-                            continue;
-                        }
-
-                        console.log(event.EventDate + '   ' + event.Severity + '   ' + event.Message);
-
-                        if (event.Message === 'Failed to deploy application.') {
-                            success = false;
-                        }
-                    }
+                for (const event of responseEvent.Events.filter(e => e.EventDate <= lastPrintedEventDate)) {
```

Comment:
1. Logs used to output in reverse order. Is this change intended?
2. the filter logic appears to be reversed (old code filtered out, new code filters in)

Reply:
1.  Hm, no: https://docs.aws.amazon.com/elasticbeanstalk/latest/api/API_DescribeEvents.html aparently it returns most recent
2. @mpiroc :P